python-sdk/pyproject.toml

```diff
@@ -91,9 +91,9 @@ openlineage = ["openlineage-airflow>=0.17.0"]
 databricks = ["databricks-cli",
     "apache-airflow-providers-databricks"]
 
-mssql = [
-    "apache-airflow-providers-microsoft-mssql>=3.2",
-]
+#mssql = [
+#    "apache-airflow-providers-microsoft-mssql>=3.2",
+#]
```

Comment:
Why are we commenting this out? Are we no longer supporting mssql?

Reply:
We're supporting, this was commented to test for local integration of Python SDK into SQL CLI. Mssql needs a few additional system dependencies to be installed which are not available yet in the runtime image. We have a ticket to address this. And I will make sure we uncomment this before merging the PR.

Reply:
https://github.com/astronomer/astro-sdk/issues/1764